exporter/tanzuobservabilityexporter/metrics.go

```diff
@@ -610,3 +610,67 @@ func (regularHistogramConsumerSpec) AsHistogram(metric pdata.Metric) histogramMe
 		HistogramDataPointSlice: aHistogram.DataPoints(),
 	}
 }
+
+type summariesConsumer struct {
+	sender   gaugeSender
+	settings component.TelemetrySettings
+}
+
+// newSummaryConsumer returns a typedMetricConsumer that consumes summary metrics
+// by sending them to tanzu observability.
+func newSummaryConsumer(
+	sender gaugeSender, settings component.TelemetrySettings,
+) typedMetricConsumer {
+	return &summariesConsumer{sender: sender, settings: settings}
+}
+
+func (s *summariesConsumer) Type() pdata.MetricDataType {
+	return pdata.MetricDataTypeSummary
+}
+
+func (s *summariesConsumer) Consume(metric pdata.Metric, errs *[]error) {
+	summary := metric.Summary()
+	summaryDataPoints := summary.DataPoints()
+	for i := 0; i < summaryDataPoints.Len(); i++ {
+		s.sendSummaryDataPoint(metric, summaryDataPoints.At(i), errs)
+	}
+}
+
+func (*summariesConsumer) PushInternalMetrics(errs *[]error) {
```

Comment:
Is this func implementing an interface? Might be nice to document it if so

Reply:
Done.